src/components/app/ProfileSharing.css

```diff
@@ -110,3 +110,13 @@
   width: 14em;
   margin: 5px -5px;
 }
+
+.profileSharingShareNetworkUrlsContainer {
+  margin: 20px 0;
```

Comment:
maybe add `-moz-user-select: none; user-select: none` to avoid the text selection but clicking repeatedly on the label.

Reply:
Added, thanks.